README.md

```diff
@@ -264,6 +267,37 @@ Each organization/tenant is associated to a different database.
 
 [Top](#top)
 
+###<a name="section3.4"></a>OrionMongoSink persistence
```

Comment:
Moreover, having the very same piece of example documentation (in this case, the mongo shell session) in both places (README.md and OrionMongoSink.md) is a bad idea. I'd suggest to include just a link to OrionMongoSink.md (and include the explanation that @gtorodelvalle suggest in that OrionMongoSink.md file).